README.md

```diff
@@ -1,14 +1,9 @@
-# Project
+# cosesign1go
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+[![go.dev](https://pkg.go.dev/badge/github.com/microsoft/cosesign1go.svg)](https://pkg.go.dev/github.com/microsoft/cosesign1go)
+[![tests](https://github.com/microsoft/cosesign1go/actions/workflows/ci.yml/badge.svg)](https://github.com/microsoft/cosesign1go/actions?query=workflow%3Aci)
```

Comment:
Just adding fancy badges on README.md.
[This](https://github.com/microsoft/cosesign1go/blob/eccbc10b29c369e59fa8f033620b41a8a1e60a48/README.md) is how it looks